.eslintrc.json

```diff
@@ -20,37 +20,30 @@
     "project": "./tsconfig.json"
   },
   "rules": {
-    "semi": ["warn", "always", { "omitLastInOneLineBlock": true }],
-    "no-multiple-empty-lines": ["warn", { "max": 2, "maxEOF": 0, "maxBOF": 0 }],
-    "quotes": ["warn", "double", { "avoidEscape": true }],
-    "camelcase": ["warn", { "properties": "always" }],
-    "curly": ["warn", "multi-or-nest"],
-    "comma-dangle": ["warn", "never"],
-    "one-var": ["warn", "consecutive"],
-    "linebreak-style": ["warn", "unix"],
+    "semi": ["error", "always", { "omitLastInOneLineBlock": true }],
+    "no-multiple-empty-lines": ["error", { "max": 2, "maxEOF": 0, "maxBOF": 0 }],
+    "quotes": ["error", "double", { "avoidEscape": true }],
+    "camelcase": ["error", { "properties": "always" }],
+    "curly": ["error", "multi-or-nest"],
+    "comma-dangle": ["error", "never"],
+    "one-var": ["error", "consecutive"],
+    "linebreak-style": ["error", "unix"],
     "prefer-const": "error",
     "no-multi-spaces": "error",
     "object-shorthand": "error",
     "quote-props": ["error", "as-needed"],
-    "prefer-destructuring": "warn",
-    "prefer-template": "warn",
+    "prefer-destructuring": ["error", { "array": false, "object": true }],
```

Comment:
why not array?

Reply:
It reduces readability when working with larger indexes and it's just annoying

Reply:
Cant we have warn for array, and error for object? Cause for like `[first, second] = []` is nice for `[, , , , idk] = []` not so

Reply:
Warnings are supposed to be fixed the same way as errors, they're just meant to be less obtrusive, so no I don't think so because that will clutter the repo and every PR going forward